tests/test_deserialization.py

```diff
@@ -741,7 +748,7 @@ def test_timedelta_field_deserialization(self):
         assert isinstance(result, dt.timedelta)
         assert result.days == 0
         assert result.seconds == 12
-        assert result.microseconds == 0
+        assert result.microseconds == 900000
```

Comment:
🤔 This was implicitly asserting that microseconds were lost. I'm not sure why, though